tests/helpers/equality-helpers.js

```diff
@@ -1,26 +1,31 @@
-export { rowValuesEqual, inputPropertiesEqual };
+export { rowValuesEqual, inputPropertiesEqual }
 
-function rowValuesEqual(assert, row) {
-  let values = Array.prototype.slice.call(arguments, 2, arguments.length);
-  let columns = row.find('th, td');
-  let columnText;
+function rowValuesEqual(...args) {
```

Comment:
Does parameter destructuring work with both arrays and rest arguments?

Like would
```js
function rowValuesEqual([expect, row, ...values]) {
```

work?